### ISSUE TITLE
stop advocating thunks

### ISSUE THREAD

**Issue by jonathanong** [tj/co#143]
with promises and async/await coming along, i think we should stop advocating thunks. having an entirely separate "co" ecosystem is kind of counter productive for the whole JS ecosystems as a whole. it'll be easier to wrap everything in promises so at least non-co people could use this libraries. `co` is somewhat a precursor to async/await as well which is promise-based, though i'll still probably still use `co` since it has a lot more features.

basically, i want to remove all the "thunkified" libraries from the wiki and primarily only allow promise-based wrappers. this means deprecating libraries like `co-fs` in favor of a promise-based wrapper like `mz`

cc @juliangruber
